lib/validator/validator.go

```diff
@@ -129,43 +138,43 @@ func validateLabels(v *validator.Validate, topStruct reflect.Value, currentStruc
 
 func validateInterface(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
 	b := []byte(field.String())
-	log.Infof("Validate interface: %s", b)
+	log.Debugf("Validate interface: %s", b)
 	return nameRegex.Match(b)
```

Comment:
I'll stick with what Felix allows - since the API needs to be at least as restrictive.